SU2_CFD/src/numerics/species/species_sources.cpp

```diff
@@ -74,10 +74,10 @@ CNumerics::ResidualType<> CSourceAxisymmetric_Species<T>::ComputeResidual(const
   AD::SetPreaccIn(Volume);
 
   if (incompressible) {
-    AD::SetPreaccIn(V_i, nDim+6);
+    AD::SetPreaccIn(V_i, idx.EddyViscosity());
```

Comment:
Don't use indices as counts please, the idea of an index instead of a hardcoded number is both readability and being able to append or insert new variables, therefore the hardcoded count is safer, and more readable because this could look like we are just registering the eddy visc
```suggestion
    AD::SetPreaccIn(V_i, idx.EddyViscosity());
```

Reply:
? I already changed this before your suggestion, don't know why this is still showing up for you. The suggested change is now the same line as what it already is in the PR...

Reply:
What I'm saying is to use nDim + 6 or +7 instead of an index... so in short reverting the change